app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -551,6 +551,7 @@ public void onMenuItemClicked(View v) {
                 Intent intent = new Intent(this, PrivateModeActivity.class);
                 startActivity(intent);
                 overridePendingTransition(R.anim.tab_transition_fade_in, R.anim.tab_transition_fade_out);
+                TelemetryWrapper.togglePrivateMode(true);
```

Comment:
Consider giving this a version so we know why the volume suddenly increases.

Reply:
added

Reply:
I think maybe it should be a new ping. SInce it's from Tab Tray